README.md

```diff
@@ -361,7 +361,9 @@ mstorage.meta("<media_id>", "user", new CompletionHandler<MediaMeta>() {
 mstorage.meta("<media_id>", "user.<key>", new CompletionHandler<MediaMeta>() {
     @Override
     public void onCompleted(Map<String, String> userMeta) {
+        String value = userMeta.get("<key>");
         // Do something.
+        // "userMeta" may have only one element with key of "<key>".
```

Comment:
I think the sentence is not necessary.
Or I think it is better than follow sentence: " ` userMeta.get("<key>")` returns the value added with the `<key>`."